app/controllers/old_exams_controller.rb

```diff
@@ -73,9 +61,7 @@ def new
 
   def edit
     @old_exam = OldExam.find(params[:id])
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unscoped call to OldExam#find](https://www.codacy.com/app/lippertsjan/klausurtool-ror/file/2128924432/issues/source?bid=3141297&fileBranchId=3143249#l63)